diff -urN orage-4.3.90.2/configure.in orage-4.3.90.2.new/configure.in
--- orage-4.3.90.2/configure.in	2006-07-10 14:35:12.000000000 +0200
+++ orage-4.3.90.2.new/configure.in	2006-09-02 23:17:01.000000000 +0200
@@ -70,7 +70,7 @@
 AC_CHECK_FUNCS([gmtime_r iswspace strdup snprintf])
 
 dnl Check for i18n support
-XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es et eu fa fi fr gu he hi hu hy it ja ko lt mr ms nl nb_NO pa pl pt_BR pt_PT ro ru sk sv ta tr uk vi zh_CN zh_TW])
+XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es et eu fa fi fr gu he hi hu hy it ja ko lt mr ms nl nb pa pl pt_BR pt ro ru sk sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for required packages
 XDT_CHECK_PACKAGE([LIBXFCE4MCS_CLIENT], [libxfce4mcs-client-1.0], [4.3.90.2])
