diff -urN orage-4.3.90.2/libical/src/libical/Makefile.am orage-4.3.90.2.new/libical/src/libical/Makefile.am
--- orage-4.3.90.2/libical/src/libical/Makefile.am	2006-07-10 14:34:44.000000000 +0200
+++ orage-4.3.90.2.new/libical/src/libical/Makefile.am	2006-09-03 20:31:43.000000000 +0200
@@ -41,6 +41,7 @@
 	-I$(srcdir)/libical		\
 	-DPACKAGE_DATA_DIR=\""$(datadir)/$(PACKAGE)"\"
 
+libical_la_LIBADD = $(PTHREAD_LIBS)
 libical_la_LDFLAGS = -no-undefined
 
 
